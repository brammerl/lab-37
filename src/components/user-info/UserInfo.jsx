import React from 'react';
import { useUser } from '../../hooks/githubContext'

const UserInfo = () => {
  const { avatar_url, followers, following, html_url, name } = useUser();
  return (
  <>
    <img src={avatar_url}/>
    <h2>{name}</h2>
    <ul>
      <h3>About</h3>
      <li>Followers: {followers}</li>
      <li>Following: {following}</li>
      <a href={html_url}>GitHub Profile</a>
    </ul>
  </>
  )
}

export default UserInfo;