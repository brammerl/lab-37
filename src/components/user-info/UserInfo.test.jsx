import React from 'react';
import { shallow } from 'enzyme';
import UserInfo from './UserInfo';


jest.mock('./UserInfo')

describe('UserInfo', () => {
  it('renders UserInfo', () => {
    const wrapper = shallow(<UserInfo/>);
    expect(wrapper).toMatchSnapshot();
  });
});