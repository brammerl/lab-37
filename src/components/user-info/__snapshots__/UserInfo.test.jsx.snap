// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfo renders UserInfo 1`] = `""`;
